content/ko/training/_index.html

```diff
@@ -0,0 +1,108 @@
+---
+title: 훈련하기
+bigheader: 쿠버네티스 교육과 인증
+abstract: 교육 프로그램, 인증 그리고 파트너
+layout: basic
+cid: training
+class: training
+---
+
+<div class="padded blue-bg">
+  <div class="main-section two-thirds-centered white-text">
+    <center>
+      <h2>당신의 클라우드 네이티브 커리어를 구축하세요</h2>
+      <p>쿠버네티스는 클라우드 네이티브 무브먼트의 핵심입니다. 리눅스 재단과 교육 파트너의 교육 그리고 인증을 통해 당신의 커리어에 투자하고, 쿠버네티스를 배우며 클라우드 네이티브 프로젝트를 성공적으로 수행할 수 있습니다.</p>
```

Comment:
```suggestion
      <p>쿠버네티스는 클라우드 네이티브 무브먼트의 핵심입니다. 리눅스 재단이 제공하는 교육과 인증 프로그램을 통해 커리어에 투자하고, 쿠버네티스를 배우며, 클라우드 네이티브 프로젝트를 성공적으로 수행하세요.</p>
```

원문은 Training and certifications이 주어지만, 교육과 인증을 받는 사람을 주어로 옮기는 것이 더 자연스러울 것 같습니다. 이 경우도 주어를 생략하는 것이 보다 자연스러울 것 같습니다. 참고 바랍니다.